#152919352 Exercise Info Endpoint

Arthur236 opened the pull request:
#### What does this PR do

Creates an endpoint in the API for viewing all information related an exercise.

#### Description of the task to be completed

There should be a new read-only API endpoint that collects all the information of an exercise such as muscles, category, images, etc. This makes it easier for clients to get all the infos since only a single request is needed to be done

#### How should this manually be tested?

After running the server with
>python manage.py runserver

, one can navigate to the following endpoint to see the results:
>http://localhost:8000/api/v2/exerciseinfo/

#### Any background context you want to provide?

#### Pivotal Tracker story

See [Feature #152919352](https://www.pivotaltracker.com/story/show/152919352)

#### Screenshots

<img width="595" alt="screen shot 2017-11-22 at 16 05 57" src="https://user-images.githubusercontent.com/31339738/33128902-31b9e450-cf9f-11e7-9b43-e13b5c8afa3f.png">

#### Questions

Arthur236 committed:
[Feature #152919352] Add serializer for exercise info

Arthur236 committed:
[Feature #152919352] Add exercise info view

Sekams committed:
fix(travis): Fix build error by gulp

- Fix Travis build error caused by the gulp command

[Fixes #152919352]

Sekams committed:
fix(api views): Remove blank line at the end

- Remove blank line at the end of the file as indicated by the pycodestyle

[Fixes #152919352]

Andretalik reviewed (changes requested):
I see some solid work, but I don't see tests for your newly added functionality.

Sekams committed:
fix(travis): Install gulp on Travis

- Install gulp cli before running Travis CI scripts

[Fixes #152919352]

Sekams committed:
test(exercise_info): Test exercise info endpoint

- Add tests for the Exercise info endpoint

[Finishes #152919352]

Sekams committed:
fix(travis): Fix Travis CI build

- Fix Travis CI build by including extra line before class declaration in test_exercise_info

[Fixes #152919352]

Andretalik commented:
Nice. I see the tests and they are passing.
